backends/gstreamer/media_stream.rs

```diff
@@ -107,9 +102,10 @@ impl GStreamerMediaStream {
             pipeline.clone()
         } else {
             let pipeline = gst::Pipeline::new("gstreamermediastream fresh pipeline");
+            let clock = gst::SystemClock::obtain();
             pipeline.set_start_time(gst::ClockTime::none());
-            pipeline.set_base_time(gst::ClockTime::from_nseconds(0));
-            pipeline.use_clock(Some(&gst::SystemClock::obtain()));
+            pipeline.set_base_time(*BACKEND_BASE_TIME);
```

Comment:
Is this change necessary? As I understand it the base times just need to match, they don't need to be the actual time things started.

To be clear: I don't want you to undo this change, I think it's a good step forward to use some semblance of a real base time, I'm more curious if `BACKEND_BASE_TIME` is really necessary (or if we can get away with using 0 everywhere)

Reply:
I'll defer the answer to @philn :)

Reply:
The base time can remain at 0 only if both pipelines start playback at the same time, which is not the case here, AFAIU? This is mentioned in the proxy docs (example): https://gstreamer.freedesktop.org/data/doc/gstreamer/head/gst-plugins-bad-plugins/html/gst-plugins-bad-plugins-proxysrc.html